service/init.go

```diff
@@ -0,0 +1,18 @@
+// TODO(ilgooz): remove this file after service package made Newable.
+package service
+
+import (
+	"log"
+
+	"github.com/mesg-foundation/core/container"
+)
+
+var defaultContainer *container.Container
+
+func init() {
+	c, err := container.New()
```

Comment:
same remark here, do we really need a state and an init function ?

Reply:
Yes, until we refactor service. This is just a temporary solution.